internal/cli/csrconfig.go

```diff
@@ -0,0 +1,135 @@
+package cli
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type CSRConfigOptions struct {
+	GlobalOptions
+	CertSigningRequestFile string
+	Outfile                string
+	Name                   string
+	ExpirationSeconds      string
```

Comment:
Why not integer?

Reply:
This processes all the data into strings for the .yaml file - it is never actually treated as an integer here